website/.prettierignore

```diff
@@ -1 +1,3 @@
 pnpm-lock.yaml
+build
+node_modules
```

Comment:
From the prettier documentation (https://prettier.io/docs/en/ignore.html)
> By default prettier ignores files in version control systems directories (".git", ".jj", ".sl", ".svn" and ".hg") and node_modules (unless the [--with-node-modules CLI option](https://prettier.io/docs/en/cli#--with-node-modules) is specified). Prettier will also follow rules specified in the ".gitignore" file if it exists in the same directory from which it is run.

As such, are these changes needed?
```suggestion
```